### ISSUE TITLE
"errors" rule. Expected RBRACE at line

### ISSUE THREAD

**Issue by alundiak** [CSSLint/csslint#603]
CSS:
![image](https://cloud.githubusercontent.com/assets/2131633/8549390/cf46a2d0-24ca-11e5-92ad-254578c63cbd.png)

and `csslint` (via grunt-contrib-csslint) with `errors: false` gives me:

```
[L47:C25]
>> ERROR: Expected RBRACE at line 47, col 25. This rule looks for recoverable syntax errors. (errors) Browsers: All
```

Is this `csslint` issue itself, or what?
I saw closed issue https://github.com/CSSLint/csslint/issues/92, very similar to this: 

Note: original spec of filter doesn't require wrapping by " ", but in fatc it fix the issue:
Having this:
```
filter: "alpha(opacity: 0)";
```
no errors.

Tested in csslint v. 0.10.0.
npm v 1.4.28
node 0.10.33
Win7.

**Comment by XOP** [comment]
+1,
disturbing issue

**Comment by OmgImAlexis** [comment]
Any updates on this?

**Comment by OmgImAlexis** [comment]
PING: @eriwen @nschonni @tomasz-oponowicz @XhmikosR @zachleat

**Comment by XhmikosR** [comment]
Yeah, no idea why it happens but I have noticed it too. This is a regression as far as I can tell.

**Comment by Duongg** [comment]
I have a similar error. I don't know what error is. Can you explain to me, please ? Thank you so much

**Comment by hakimio** [comment]
Still an issue. Any plans to fix it?
@eriwen @nschonni @tomasz-oponowicz @XhmikosR @zachleat

**Comment by lulu5239** [comment]
Il manque un ; à la fin de la ligne du dessus.
It's missing a ; at the end of the top line.

**Comment by HYR** [comment]
I'm seeing the same issue as @hakimio. Hope it gets resolved soon.

**Comment by Showcase-Joz** [comment]
yup, really annoying when you spend a while ensuring there are no code errors 👎

**Comment by dkhalife** [comment]
Any updates on this? It seems to also break on `:global` in the code snipped below:

```
:global(.myclass) hr {
    margin: 10px;
}
```

**Comment by BradHeff** [comment]
Im getting the same thing

> CSSLint: Expected RBRACE at ......

`:root {
  --blue: #007bff;
}`

**Comment by manuel-martinez-dev** [comment]
> Im getting the same thing
> 
> > CSSLint: Expected RBRACE at ......
> 
> `:root { --blue: #007bff; }`

I get the same error too... also CSSLint is saying that the rule (:root) is empty...

**Comment by GeneralMine** [comment]
Still an issue in 2021 and linked to #720
